ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1_FABRIC.yml

```diff
@@ -0,0 +1,89 @@
+---
+### group_vars/DC1_FABRIC.yml
+
+# Set the Fabric Name - must match an Ansible Inventroy Group
+fabric_name: DC1_FABRIC
+
+# Set Design Type to L2ls
+design:
+  type: l2ls
+
+# Ansible connectivity definitions
+# eAPI connectivity via HTTPS is specified (as opposed to CLI via SSH)
+ansible_connection: ansible.netcommon.httpapi
+# Specifies that we are indeed using Arista EOS
+ansible_network_os: arista.eos.eos
+# This user/password must exist on the switches to enable Ansible access
+ansible_user: admin
+ansible_password: admin
+# User escalation (to enter enable mode)
+ansible_become: true
+ansible_become_method: enable
+# Use SSL (HTTPS)
+ansible_httpapi_use_ssl: true
+# Do not try to validate certs
+ansible_httpapi_validate_certs: false
+
+
+## set to none for pure L2LS - no routing or svis
+# underlay_routing_protocol: none
```

Comment:
Do we need this?

Reply:
how are about for the l3 modification, don't we need to set a routing protocol?

Reply:
My suggestion is to remove this comment all together.  It is not needed in this example as we are not discussing routing protocols.